src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -618,6 +625,48 @@ private static ITaskSeriesTimer StartParameterLogTimer(IRecurrentCommand updateC
             //      a. If throwOnTimeout, we throw the FunctionTimeoutException.
             //      b. If !throwOnTimeout, wait for the task to complete.
 
+            // Where the invocation filter attribute functionality is currently temporarily placed
+            var functionMethod = instance.FunctionDescriptor.Method;
+            var invokeFilter = functionMethod.GetCustomAttribute<InvocationFilterAttribute>();
```

Comment:
Take a look at the existing TypeUtility.GetHierarchicalAttributeOrNull method, which walks up the hierarchy and returns the first attribute found.

You should add a new method to TypeUtility that does this walk and returns multiple attributes. E.g. name it GetHierarchicalAttributes. It walks up from the method and returns all Method/Class level attributes.